src/rfbzero/experiment.py

```diff
@@ -379,11 +480,12 @@ def run(
 
     @staticmethod
     def _validate_cycle_values(
-            value: float,
-            value_charge: float,
-            value_discharge: float,
+            value: float | None,
```

Comment:
Note that if we're using union types, our supported Python version will be 3.10+

Reply:
>=3.10 is enforced